src/commands/kick.ts

```diff
@@ -0,0 +1,31 @@
+import Command from "../command";
+import * as DiscordJS from "discord.js";
+import * as VoteSystem from "../vote-system";
+
+export = new Command
+(
+	"kick",
+
+	async (message: DiscordJS.Message) =>
+	{
+		if (VoteSystem.Poll.currentPoll === null)
+		{
+			VoteSystem.Poll.currentPoll = VoteSystem.Poll.startPoll(message, "kick", (member: DiscordJS.GuildMember) => member.kick("This person was voted to be kicked"), (2 / 3), false);
```

Comment:
We can check here to make sure that more than half of the users on the server are online before starting the poll. Don't worry about code duplication for now, I'll fix it later.

Reply:
I was thinking it would be fine if the poll was started with not too many players online, because of the required amount of users to upvote in the poll, they would have to wait anyway, it makes it so some people can vote while they are online.

Reply:
I agree that this would be nice, but with the current poll system, only one poll can be underway at a time. If we left the poll running all night, no one would be able to start a new one, even if it was for something simple like an unmute.

We can also fix this with timeouts, but for now let's require half of the server to be online before starting. Feel free to leave a comment here so that we remember to fix it once the poll system is ready for it.

Reply:
Oh, forgot about that limitation, makes more sense